src/vm/util.cpp

```diff
@@ -2751,7 +2751,91 @@ HMODULE CLRLoadLibrary(LPCWSTR lpLibFileName)
     return hmod;
 }
 
-#ifndef FEATURE_PAL
+#ifdef FEATURE_PAL
+
+static HMODULE CLRLoadLibraryDirectWorker(LPCWSTR lpLibFileName, DWORD *pLastError)
```

Comment:
You do not need this wrapper code for PAL - just call the PAL LoadLibraryDirectW directly.

ThreadAffinityHolder / SetErrorMode are no-ops for CoreCLR / PAL.